mmda/types/names.py

```diff
@@ -6,5 +6,13 @@
 
 """
 
+# document field names
 Symbols = 'symbols'
 Images = 'images'
+
+Pages = 'pages'
```

Comment:
I have a question on this. We only create the fields for all the default fields that the document parser can generate, right? But we also allow users create their own fields by the annotate function, yet they can't change the fields here in their code. Would it be a bit weird? 

And FYI, we could also try something like this: https://github.com/huggingface/transformers/blob/24cbf6bc5a0b6a9bb5afdda6bb1a329ac980fa4b/src/transformers/file_utils.py#L1941

Reply:
not that weird.  the purpose of this is simply to keep the library clean and without string name typos.

if someone wants to add a model to the library that populates a particular field, they can include in their PR the name of that field in this file.  

if they are adding fields in their own python sessions, then it's not our job to manage those names